### PR TITLE
Belt.HashMap examples

### LINKED ISSUE #52: Create examples for every function of HashMap
https://github.com/reason-association/reasonml.org/blob/master/pages/belt_docs/hash-map.mdx

### PR DESCRIPTION
**Pull request by baransu**
This PR adds examples (resolve #52 ) to following modules.
- [x] Belt.HashMap
- [x] Belt.HashMap.String
- [x] Belt.HashMap.Int

### CODE REVIEW

**Comment by baransu** [comment]
@ryyppy Not sure why now preview deployment is failing 😕

**Comment by baransu** [comment]
@ryyppy Now still is failing.

File: yarn.lock [hunk 1]
```diff
@@ -1146,11 +1146,6 @@
     mini-css-extract-plugin "0.4.3"
```
Comment (ryyppy) [inline yarn.lock]: Is it necessary to update the yarn.lock file?

Comment (baransu) [reply yarn.lock]: Don't need that. Just a result of me running `yarn install` with the `yarn 1.22`.

**Comment by ryyppy** [comment]
it's a problem with the now.json configuration. I will merge it, thanks for the contribution :)

**Pull request merged**